Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/BaseController.java

```diff
@@ -223,14 +223,18 @@ protected String getExportPath(final String nativeType) {
 
     public void toggleComponentSclass(HtmlBasedComponent comp, boolean state, String stateOff, String stateOn) {
         String sclass = Objects.requireNonNull(comp.getSclass(), stateOff);
```

Comment:
Is it better if we just handle null value instead of throwing NullPointerException for this method?
if (comp) == null return;
if (comp.getSclass() == null) sclass = ""

Reply:
In the same vein of input validation above, we can also add handling for stateOn and stateOff because NullPointerException will be thrown in String.contains. NullPointerException will lead to unexpected error message from the system.
if (stateOn == null || stateOff == null) return;